styles.css

```diff
@@ -22,48 +28,192 @@ h1 {
+  color: #1a2236;
+  border: 1px solid #1a2236;
+}
+
+#linkbox:active {
+  background-color: #fff;
+  color: #1a2236;
+  border: 1.5px solid #1a2236;
+  outline-style: solid;
 }
 
 footer {
-  height: 240px;
-  width: 100%;
-  background-image: url("image_geometry_2.svg");
+  height: 250px;
+  width: 100vw;
+  background-image: url("images/image_geometry_2.svg");
   background-repeat: no-repeat;
   background-size: cover;
   margin-top: 32px;
 }
 
-.alignleft {
-  text-align: left;
+h2 {
+  font-family: 'Playfair Display', serif;
+  font-style: normal;
+  font-size: 50px;
+  color: #1a2236;
+  width: 220px;
+  font-weight: 400;
+  line-height: 100%;
+  margin: 40px 16px 32px;
+}
+
+#project {
+  display: grid;
+  gap: 0;
+  grid-template-rows: 290px 220px;
+}
+
+#projectimage {
+  width: 100vw;
+}
+
+#projectdetails {
+  display: flex;
+  flex-direction: column;
+  background-color: #1a2236;
+  border: 0.333333px solid rgba(255, 255, 255, 0.1);
+  backdrop-filter: blur(0.5px);
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+}
+
+#projectname {
+  height: 16px;
+  margin: 20px 36px 20px 16px;
+  font-weight: 300;
+  font-size: 20px;
+  line-height: 16px;
+}
+
+#projectdetails ul {
+  display: grid;
+  gap: 10px;
+  grid-template-columns: auto auto auto;
+  justify-content: start;
+  margin: 16px auto 0 20px;
+}
+
+#projectdetails li {
+  height: 20;
+  background: #98dede;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 10.3334px;
+  line-height: 8px;
+  color: #1a2236;
+  border: 1px solid black;
+  text-align: center;
+  padding: 6px 10px;
+}
+
+#projectlink {
+  display: flex;
+  flex-wrap: nowrap;
+  border: 2px solid #fff;
+  margin: 20px auto 20px 20px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  padding: 10px 12px 10px 10px;
+  justify-content: space-around;
+}
+
+#union {
+  margin-left: 12px;
+}
+
+#aboutmyself {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+}
+
+#downloadlink {
+  font-family: 'Poppins', sans-serif;
+  margin: 32px 16px 40px 0;
+  font-style: normal;
+  color: #fff;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  background-color: #1a2236;
+  height: 50px;
+}
+
+#ic_download {
+  margin-left: 20px;
+  margin-top: 6px;
+}
+
+#aboutmyself section {
+  display: flex;
+  flex-direction: column;
+  height: 450px;
+  width: 450px;
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+}
+
+#aboutmyself ul {
+  padding-left: 18px;
+}
+
+#languages {
+  background: #1c528a;
+  background-image: url("images/languagesimage.svg");
+}
+
+#frameworks {
+  background: #e2505c;
+  background-image: url("images/ic_framework.png");
+}
+
+#skills {
+  background: #febf52;
+  background-image: url("images/ic_skills.svg");
 }
```

Comment:
- [ ] Good job by implementing the language section but you still have some issue to fix with your design. Between the languages there is space, please kindly add the padding between your languages. Your design should stick as much as possible to the Figma design 🙃 🙃

Your design

![Screen Shot 2022-10-01 at 20 25 12](https://user-images.githubusercontent.com/74501523/193423405-3d35864b-b566-488d-8272-d8a5e3965599.png)

Figma design

![Screen Shot 2022-10-01 at 20 24 36](https://user-images.githubusercontent.com/74501523/193423417-c42f798a-52f7-4bf7-8c58-319bb6acd356.png)

Reply:
design modified to match the figma